### PR TITLE
API endpoint to get channel members by IDs.

### PR DESCRIPTION
**Pull request by grundleborg**
The purpose of this is to make it possible to find out the roles of the members of a channel.

This will partially enable PLT-5049.

This is designed to be analogous to the Team Members By Ids API endpoint.

### CODE REVIEW

File: source/channels.yaml [hunk 1]
```diff
@@ -343,6 +343,39 @@
...
+  '/teams/{team_id}/channels/{channel_id}/members/ids':
+      post:
+        tags:
+          - channels
+        summary: Get channel members by IDs
+        description: Get a map of channel member objects for the specified channel and user IDs.
+        parameters:
+          - name: team_id
+            in: path
+            description: Team ID of the member to return
```
Comment (jwilander) [inline source/channels.yaml]: Maybe just change this to `Team ID for the corresponding channel` or something similar?

File: source/channels.yaml [hunk 2]
```diff
@@ -343,6 +343,39 @@
...
+            required: true
+            schema:
+              type: array
+              items:
+                type: string
+        responses:
+          '200':
+            description: Members retreived successfully
+            schema:
+              type: object
```
Comment (enahum) [inline source/channels.yaml]: does this return a map or an array?

Comment (jwilander) [inline source/channels.yaml]: Map is consistent with the similar endpoint for team members

Comment (grundleborg) [inline source/channels.yaml]: Good question! It looks like the equivalent method for TeamMembers returns an Array in the code, but claims to return a Map in the API reference.

I'll change this one to an array too, unless you think it makes more sense to change things to a map here?

Comment (jwilander) [inline source/channels.yaml]: Oh, if it's array in code then yeah let's switch it over for this and can you also update the documentation for the team members one to be accurate?

**Pull request merged**